doc/usage.rst

```diff
@@ -89,6 +89,30 @@ Or in Windows, use the little-known BAT file line continuation::
         myscript.spec
 
 
+Running PyInstaller with Python optimizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. Note::
+
+	You should be aware of the ``PYTHONOPTIMIZE`` mechanism when using
```

Comment:
* Well, this is not an "option" for pyinstaller, so I suggest you using different wording.
* I'd start the sentence with the "When using this …"
* "be aware of" sound like: you should know there is such a think. But one needs no know what this does.
* ``PYTHONOPTIMIZE`` is the name of the env-var. Maybe better just "Python Bytecode Optimization mechanism"

Suggestion:
> When using this feature/option/thing, you should be aware of how the the ``PYTHONOPTIMIZE`` mechanism works.